### ISSUE TITLE
Invalid arguments: "timeout" must be a number

### ISSUE THREAD

**Issue by Edsase** [plotly/falcon-sql-client#388]
Connection timeout field displays the above error even when the input is a number (both integer and float raise this error)

**Comment by shannonlal** [comment]
Which db connector are you using?

**Comment by Edsase** [comment]
SQLServer

**Comment by shannonlal** [comment]
Have you been able to reproduce this with the sample data provided.  For example if you click on the SQLServer connector and then click on Sample Credentials.  Are you able to reproduce this error?  Can you provide the information you used.  It would help us test to determine what the issue is

**Comment by Edsase** [comment]
I think i didn't describe the error well. Let me try again:
I am trying to connect to an SQLServer which generally takes a while to load even with other clients. So Falcons default  connection timeout of 15000milliseconds is too short. I tried to change this field using increment arrow and manual input. Both approaches lead to the error "Invalid arguments: "timeout" must be a number".

**Comment by n-riesco** [comment]
@Edsase I can't reproduce this issue (I even tried to edit `%userprofile%\.plotly\connector\connection.yaml` manually).

What value of `connectTimeout` are you using?

Did you get to save the connection? Assuming that you are on windows, a list of saved connections can be found in `%userprofile%\.plotly\connector\connection.yaml` (this a text file that can be opened with notepad).

**Comment by n-riesco** [comment]
Closing since it can't be reproduced and there hasn't been any recent activity.
